force-app/main/default/flows/AssignNextStep.flow-meta.xml

```diff
@@ -0,0 +1,50 @@
+<?xml version="1.0" encoding="UTF-8"?>
```

Comment:
Can we achieve tha goal without using flow. and don't create anything manually if it exists already.